lavamoat/browserify/desktop/policy.json

```diff
@@ -1,4614 +1,4698 @@
 {
-    "resources": {
```

Comment:
Interesting, it looks like this had a 4-sp;ace indentation rather than two. That is why this diff is so huge; `lavamoat auto` made it use 2-space indentation